packages/docusaurus-theme-classic/src/theme/Layout/styles.css

```diff
@@ -5,8 +5,23 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+html,
+body {
+  height: 100%;
+}
+
 body {
   margin: 0;
   padding-top: var(--ifm-navbar-height);
   transition: var(--ifm-transition-fast) ease color;
 }
+
+body > div {
```

Comment:
It's better not to use tags directly (except for html/body)

Reply:
Could we add these styles to `body` instead?

Reply:
> Could we add these styles to body instead?

Nope, we need to apply styles to the root element, i.e. id="__docusaurus", but it seemed like a bad use to me (this name can change, but here `body > div` applies just to the root element, which will always be one in the `body` tag.

The same goes for the Bootstrap approach.

Reply:
By the way, as for the Bootstrap approach - there the styles are also applied to the footer, which can be hidden (`noFooter`), then CSS rules for it needs to be defined in its component - Footer, not Layout. I want to say that the current approach is more integral - the change is only in Layout.